pkg/runtime/k3s/bootstrap.go

```diff
@@ -193,6 +230,14 @@ func (k *K3s) pullKubeConfigFromMaster0() error {
 
 func (k *K3s) copyKubeConfigFileToNodes(hosts ...string) error {
 	src := k.pathResolver.AdminFile()
+	data, err := file.ReadAll(src)
+	if err != nil {
+		return errors.WithMessage(err, "read admin.config file failed")
+	}
+	newData := strings.ReplaceAll(string(data), "https://0.0.0.0:6443", fmt.Sprintf("https://%s:%d", constants.DefaultAPIServerDomain, 6443))
```

Comment:
if user have specified the `HTTPSPort`, then we should call `getAPIServerPort` but not use 6443 directly.